content/en/docs/guide_openapi/managing_apidocs.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Managing OpenAPI documents
+weight: 30
+date: 2022-03-01
+description: This document contains details about OpenAPI specification support, and the management of the OpenAPI document lifecycle, from importing, upgrading revisions, to deleting them.
+---
+
+## OpenAPI specification support
+
+Currently, **OpenAPI** flow-trigger supports the following specifications:
+
+* [OpenAPI 2.0](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/2.0.md)
+* [OpenAPI 3.0](https://github.com/OAI/OpenAPI-Specification/blob/main/versions/3.0.3.md)
+
+If you do not already have an OpenAPI specification to manage, you can read our guide on [how to write OpenAPI documents](/docs/guide_openapi/writing_apidocs).
+
+## Importing OpenAPI documents
+
+Currently, only one **OpenAPI** document is supported for **OpenAPI** flow-trigger. Once loaded, you can [update](#updating-openapi-documents) or [delete](#deleting-openapi-documents) it as required.
```

Comment:
Can import multiple now. Need to figure out what to say, and probably have some mention of how multiple docs are managed. Somewhere in this section we also need to mention how the filename is important and influences where the docs are hosted, and link to the section in the quick start page (which also needs updating)

Reply:
Need to check the rest of the docs for references to the previous single spec behavior.